src/Polytopes/Cone/properties.jl

```diff
@@ -161,7 +161,7 @@ isfulldimensional(C::Cone) = pm_cone(C).FULL_DIM
 ## Points properties
 ###############################################################################
 
-"""
+@doc Markdown.doc"""
```

Comment:
There seems to be a lot in this PR which doesn't have anything to do with graphs?